airgun/entities/subscription.py

```diff
@@ -139,6 +139,19 @@ def read_subscription_manifest_expiration_date_only(self):
             manage_view.close_button.click()
         return manifest_expiration_date
 
+    def is_subscription_manifest_header_message_display(self):
+        """Is header & message present in manage manifest modal box or not"""
+        result = False
+        view = self.navigate_to(self, 'Manage Manifest')
+        view.wait_animation_end()
+        if view.manifest.alert_message.is_displayed:
+            result = True
+        # close opened modal dialogs views
+        manage_view = ManageManifestView(self.browser)
+        if manage_view.is_displayed:
+            manage_view.close_button.click()
+        return result
```

Comment:
Is it possible to return the actual message if displayed, False if absent? 
Or do you think it is better as a strictly boolean return type?

Reply:
there is another function which is returning 3 things (header, message & date) if you check `read_subscription_manifest_header_message_and_date`. This is just to check presence/absence of header message, we want only boolean value out of it. I would like to keep the behaviour as it is.